config/global-constants/src/lib.rs

```diff
@@ -11,6 +11,7 @@
 /// Definitions of global cryptographic keys (e.g., as held in secure storage)
 pub const ASSOCIATION_KEY: &str = "association";
 pub const CONSENSUS_KEY: &str = "consensus";
+pub const EXECUTION_KEY: &str = "execution";
```

Comment:
let's update any relevant documentation about this as well as @mgorven

Reply:
What documentation you can think of?